synapse/handlers/initial_sync.py

```diff
@@ -165,7 +165,11 @@ async def handle_room(event: RoomsForUser) -> None:
 
                 invite_event = await self.store.get_event(event.event_id)
                 d["invite"] = await self._event_serializer.serialize_event(
-                    invite_event, time_now, as_client_event
+                    invite_event,
+                    time_now,
+                    # No need to bundle aggregations for state events.
+                    bundle_aggregations=False,
+                    as_client_event=as_client_event,
```

Comment:
This potentially fixes an unrelated bug -- `as_client_event` was being passed in as `bundle_aggregations` instead of being passed through as a `kwarg` so `as_client_event` would always end up being `True` (the default).

Reply:
ooof. maybe `bundle_relations` should be a kw-only arg to catch these problems earlier.

Reply:
I think that makes sense. I can make that change.

Reply:
I added this in 2f409daa8cb1834191bd5632ad3feab1584846df.